NeuroID/src/advancedDeviceAndroidLib/java/com/neuroid/tracker/service/LoggerIntercepter.kt

```diff
@@ -0,0 +1,21 @@
+package com.neuroid.tracker.service
+
+import com.neuroid.tracker.utils.NIDLogWrapper
+import okhttp3.Interceptor
+import okhttp3.Response
+
+class LoggerIntercepter(val logger: NIDLogWrapper): Interceptor {
+
+    override fun intercept(chain: Interceptor.Chain): Response {
+        val request = chain.request()
+        val response = chain.proceed(request)
+        if (response.code() != 200) {
+            logger.e("Neuro ID","http error, ${request.method()}, ${request.url()}, " +
```

Comment:
I don't think we need the tag anymore because of Priya's work. if you want to keep it lets do one word `NeuroID`

Reply:
removed tag